portfolio.html

```diff
@@ -9,56 +9,41 @@
     <link href="https://fonts.googleapis.com/css?family=Josefin+Sans" rel="stylesheet">
 </head>
 <body>
-<main>
-    <div class="outer-container">
+
+    <div class="container">
         <h1>Portfolio</h1>
-        <div class="inner-container">
-            <a href="#">
-                <div class="col-4">
-                    <img class="portfolio-image">
-                </div>
-            </a>
-            <a href="#">
-                <div class="col-4">
-                    <img class="portfolio-image">
-                </div>
-            </a>
-            <a href="#">
-                <div class="col-4">
-                    <img class="portfolio-image">
-                </div>
-            </a>
-            <a href="#">
-                <div class="col-4">
-                    <img class="portfolio-image">
-                </div>
-            </a>
-            <a href="#">
-                <div class="col-4">
-                    <img class="portfolio-image">
-                </div>
-            </a>
-            <a href="#">
-                <div class="col-4">
-                    <img class="portfolio-image">
-                </div>
-            </a>
-            <a href="#">
-                <div class="col-4">
-                    <img class="portfolio-image">
-                </div>
-            </a>
-            <a href="#">
-                <div class="col-4">
-                    <img class="portfolio-image">
-                </div>
-            </a>
+        <div class="gallery">
```

Comment:
Might be worth considering a semantic tag here instead of div.

Reply:
I'm ok with it. Checked with Charlie.